index.js

```diff
@@ -29,80 +33,25 @@ function create_mainWindow() {
     return mainWindow;
 }
 
-var menuBar = [
-    {
-        label: 'App',
-        submenu: [
-            {
-                label: 'Quit',
-                accelerator: 'ctrl+q',
-                click() {
-                    app.quit();
-                }
-            },
-            {
-                type: 'separator'
-            },
-            {
-                label: 'Reload',
-                accelerator: 'ctrl+r',
-                click() {
-                    mainWindow.webContents.reload();
-                }
-            },
-            {
-                type: 'separator'
-            },
-            {
-                label: 'Force Reload',
-                accelerator: 'ctrl+f',
-                click() {
-                    mainWindow.webContents.reload(true);
-                }
-            },
-        ]
-    },
-    {
-        label: 'Developers',
-        submenu: [
-            {
-                label: 'Open DevTools',
-                accelerator: 'ctrl+s',
-                click() {
-                    mainWindow.webContents.openDevTools();
-                }
-            }
-        ]
-    },
-    {
-        label: 'About',
-        submenu: [
-            {
-                label: 'Git Repository',
-                async click() {
-                    opn('https://github.com/Zeal8bit/Zeal-WebEmulator');
-                }
-            },
-            {
-                label: 'Version',
-                click() {
-                    console.error("Version window is developing");
-                }
-            }
-        ]
-    },
-];
-
-const createWindow = () => {
-
+function createWindow() {
     mainWindow = create_mainWindow();
     // Set up the menu bar
     const menu = Menu.buildFromTemplate(menuBar);
     // Set up the top menu
     Menu.setApplicationMenu(menu);
 };
 
-app.on('ready', createWindow);
+app.on('ready', () => {
+    let argv = getArgs();
+    if(!argv) {
```

Comment:
we shouldn't quit just because there we no args passed?

Reply:
true, but why exit if it's not? the code that parses the args should be able to handle a null/undefined argv if it's not available, using whatever defaults are available.  This just seems like added complexity, with no benefit?